src/main/java/Duke.java

```diff
@@ -1,10 +1,246 @@
+import duke.command.AException;
+import duke.command.Parser;
+import duke.command.Storage;
+import duke.command.TaskList;
+import duke.command.Ui;
```

Comment:
You may want to consider creating another separate package for `Ui` and moving `TaskList` class  to `duke.tasks` package.